yk_format_new_files.sh

```diff
@@ -4,8 +4,4 @@
 
 set -e
 
-# The LLVM version tag we are forked from.
-FORKED_LLVM_VERSION=`git describe --abbrev=0 --match 'llvmorg-*'`
-
-git diff --name-only --diff-filter=A ${FORKED_LLVM_VERSION} \
-    `git branch --show-current` | egrep '\.(cpp|h)$' | xargs clang-format -i
+clang/tools/clang-format/git-clang-format llvm/main
```

Comment:
Will this format everything? Is that what we want?

Reply:
Only lines differing from `llvm/main`

Reply:
OK.

Reply:
It should only format anything that is new since `llvm/main`. It's the same command I used to do https://github.com/ykjit/ykllvm/pull/37/commits/547b537189934e5f5921cc0c7664df56fd94404d